microsoft-edge/webview2/how-to/debug-visual-studio.md

```diff
@@ -6,98 +6,181 @@ ms.author: msedgedevrel
+
+   ![Running the Local Windows Debugger in Visual Studio](./debug-visual-studio-images/run-local-windows-debugger.png)
+
+   When setting a breakpoint, you must open the file that's in the exact same path that's associated with your WebView2 control, because the JS/TS debug adapter doesn't perform source path mapping.  
 
-   ![Adding a breakpoint in Visual Studio](./debug-visual-studio-images/breakpoint.png)
+   The breakpoint will be triggered when this section of the code is executed.  This breakpoint is triggered upon clicking the **Add a new item** button in the sample app, in a later step below.
 
-1. To run the debugger, select the bit size of the platform, and then click the green Play button next to **Local Windows Debugger**.  The app runs and the debugger connects to the first WebView2 process that is created.
+1. Select the bit size of the platform, such as **x64**.
 
-   ![The Local Windows Debugger in Visual Studio](./debug-visual-studio-images/run.png)
+1. To run the debugger, do any of the following: 
 
-1. In the **Debug Console**, find the output from the debugger.
+   * In the menu bar, select **Debug** > **Start Debugging (F5)**.
+   * In Solution Explorer, right-click the project > **Debug** > **Start New Instance** or **Step Into New Instance**.
+   * Click the green Play button to the left of **Local Windows Debugger**.
+    
+   The WebView2APISample app (or your own app) runs and opens, and the debugger connects to the first WebView2 process that's created.  In **Properties** > **Debugging** for the project, **Local Windows Debugger** was set to **JavaScript (WebView2)**.
 
-   ![Debug Console in Visual Studio](./debug-visual-studio-images/console.png)
+1. In Visual Studio, click the **Output** tab, in the lower right.
+
+   The tab contains only the following message, which mentions the JS debugger: "Verbose logs are written to: C:\Users\myUsername\AppData\Local\Temp\visualstudio-js-debugger.txt".
+
+1. In the WebView2APISample app window, select **Scenario** > **Script Debugging** > **JavaScript Local File**.
+
+   The file `WebView2APISample/Debug/x64/assets/ScenarioJavaScriptDebugIndex.html` is rendered:
+
+   ![Rendered HTML file "WebView Script Debugging Example"](./debug-visual-studio-images/webview-script-debugging-example.png)
+
+1. In Visual Studio, examine the **Output** pane.  The following output has been added:
+
+   ```
+   This is the very first line of code that executes.
+   Second
+   Third
+   inside
+   onAddClick+
+   onAddClick-
+   onAddClick+
+   onAddClick-
+   End
+   ```
+
+1. In the webpage in the app, click the **Add a new item** popup button.
+
+   The breakpoint in the `onAddClick` function is hit, and the app is paused in the debugger.
+
+1. Switch to the Visual Studio window, and examine the debugger panes.
```

Comment:
When walking through the end-to-end steps in revised article, I never did get the debugger to halt at breakpoint, after I clicked Add Item in the running wv2 sample app.  
@captainbrosset 
 
Nice to do: Patrick do the steps in PR's article, see if hits breakpoint.  See if article gives the usage info that the article ideally should give, about why we're setting a breakpoint, and what exactly eg we can do in the debugger panes (I never arrived at useful debugger panes UX).

Reply:
I did the steps from the tutorial, but couldn't get the breakpoint to hit either. It looks like we're maybe missing some configuration?
Here's what I'm seeing when setting a breakpoint:

![image](https://github.com/user-attachments/assets/d0423510-4437-4d0a-a572-46ba12af9a0f)

Reply:
i still suspect, check the settings in
https://review.learn.microsoft.com/en-us/microsoft-edge/webview2/how-to/debug-visual-studio-images/enb-js.png
a lot going on there, that could affect UX.
@captainbrosset 
seems like need real, actual explanation of what's going on: we're installing another debugger, and trying to set this debugger to be used, for some scenario.  that background explanation is proved to be needed & missing here.

Reply:
Lets use AB#54124902 to track this.